File: webtracksystem/web/components/devices/views/code.js
import React, { PropTypes } from 'react';

const DeviceCode = ({device}) => (
  <div className="root">
    {device && <section className="">
        <h2 className="title">Device Auth Code</h2>
        <div>{device.imei}</div>
        <div>{device.title}</div>
        <div>{device.code}</div>
    </section>}
  </div>
);

DeviceCode.propTypes = {
    //code: PropTypes.func.isRequired
};

export default DeviceCode;

File: webtracksystem/businesslayer/Models/Device/Code.cs

using System;

namespace BusinessLayer.Models
{
    public class DeviceCode
    {
        public int Id { get; set; }
        public string IMEI { get; set; }
        public int Code { get; set; }
        public DateTime Expiration { get; set; }
        public Device Device { get; set; }
    }
}